### ISSUE TITLE
Is this still active

### ISSUE THREAD

**Issue by Popesites** [enigmampc/SecretJS-Templates#24]
Last commit was 6 months ago, is this still maintained / current with Secret?

**Comment by levackt** [comment]
Hi @Popesites , yes until the mainnet upgrade yesterday all the samples were working.

Just now confirmed that [my issue ](https://github.com/scrtlabs/SecretJS-Templates/pull/21) persists with the latest version of SecretJS, ie the contract example 5 is not working. I'm look into that to get it current again, and perhaps @assafmo has some idea once he has a chance to check it out too.

Either way I'll update here to close this issue once it's working, and likely also have to update the Figment Learn examples.

**Comment by monomadic** [comment]
None of these examples work any more. I mean this constructively - it's not a great look for coming to SecretNetwork as a dev and finding the official examples completely outdated and broken.

**Comment by assafmo** [comment]
You're right. There's a partially working branch [`update-secretjs`](https://github.com/scrtlabs/SecretJS-Templates/tree/update-secretjs). Only steps 1-4 are working there, I need to complete 5-8.

In the meantime you can go through 1-4 and it would be awesome to get some help from the community here to complete this. Otherwise I'll try to get to it soon.

**Comment by levackt** [comment]
> You're right. There's a partially working branch [`update-secretjs`](https://github.com/scrtlabs/SecretJS-Templates/tree/update-secretjs). Only steps 1-4 are working there, I need to complete 5-8.
> 
> In the meantime you can go through 1-4 and it would be awesome to get some help from the community here to complete this. Otherwise I'll try to get to it soon.

Picking this up @assafmo

**Comment by monomadic** [comment]
Steps 1-4 don't work btw. Node endpoints need updating also. Do you need help on this?

**Comment by levackt** [comment]
> Steps 1-4 don't work btw. Node endpoints need updating also. Do you need help on this?

help is always welcome, but I now have 1 through 5 working in the branch I just PR'd into @assafmo 's WIP

See if that works for you as well.

Locally I have websockets and some of the keplr stuff working too, fighting a little with keplr so will backoff and circle back

**Comment by levackt** [comment]
Just updated for 7 and 8 as well, that only leaves keplr if you get there before me.
Oh and multi messages for contracts but I don't know if that can still work in the same way, might end up as a new example.
Hope that helps you get started in the meantime

**Comment by monomadic** [comment]
Thanks, I'll try these in the morning and let you know.

**Comment by levackt** [comment]
FYI @monomadic , I just pushed the original Keplr / reactjs example, so we now have one for balances and another for contracts executions.

Also switched the old multiExecute for broadcast, for multiple contract executions, or any arbitrary messages.

The only one still giving me a bit of grief is the vanilla JS Keplr
